schema/transportation/segment.yaml

```diff
@@ -221,6 +221,31 @@ properties:
                     items: { "$ref": "#/$defs/propertyDefinitions/lane" }
                     minItems: 1
               minItems: 1
+        signposts:
+          description: Describes signpost along the a segment
+          type: array
+          items:
+             allOf:
+               - "$ref": "#/$defs/propertyContainers/directionContainer"
+             unevaluatedProperties: false
+             type: object
+             properties:
+               destination:
+                 type: object
+                 required: [segmentId, signTextType, signText]
+                 allOf:
+                   - "$ref": "#/$defs/propertyContainers/directionContainer"
+                 properties:
+                   segmentId: 
+                     type: string
+                   connectorId: 
+                     type: string # Connector id is optional if the segments are not connected or if the the connector can be unambiguously identified 
+                   signDirection:
+                     "$ref": "#/$defs/propertyDefinitions/compassDirection"
+                   signTextType:
+                     "$ref": "#/$defs/propertyDefinitions/signTextType"
+                   signText: 
```

Comment:
- The "stutter naming" (repetition of the prefix word "sign") doesn't seem necessary here. Can't `signDirection` be `direction`, *etc.*?
- I notice this doesn't say `signpostDirection`, *etc.* This seems to support my contention that the important thing is the sign, not the post holding it up :)
- Where are we going to get the data for this? How much of it exists in OSM, in which tags?